src/hermezjs-typings.d.ts

```diff
@@ -405,7 +406,14 @@ declare module "@hermeznetwork/hermezjs/src/tx" {
   ): Promise<TxData>;
 
   // function withdraw();
-  // function isInstantWithdrawalAllowed();
+  function isInstantWithdrawalAllowed(
+    amount: string,
+    accountIndex: string,
+    token: Token,
+    babyJubJub: string,
+    batchNum?: number,
+    merkleProofSiblings?: MerkleProof["siblings"]
+  ): Promise<unknown>;
```

Comment:
it's returning a empty array at this moment, maybe I can change it to unknown[]. but we don't use it, just if we catch an error

Reply:
But an empty array of what? Don't we know?

Reply:
Ok, checking `hermezjs` I see that `isInstantWithdrawalAllowed` does a call to the function `withdrawMerkleProof` of the contract. According to the types I generated with [`TypeChain`](https://github.com/dethcrypto/TypeChain) the signature of this function is as follows:

```typescript
withdrawMerkleProof(
  tokenID: BigNumberish,
  amount: BigNumberish,
  babyPubKey: BigNumberish,
  numExitRoot: BigNumberish,
  siblings: BigNumberish[],
  idx: BigNumberish,
  instantWithdraw: boolean,
  overrides?: Overrides & {from?: string | Promise<string>}
): Promise<ContractTransaction>;
``` 

But since we don't have the contract typed I agree that for now `unknown` or `Promise<unknown>` are the best option.